### PR TITLE
Add myget pre-built binaries info on README

### PR DESCRIPTION
**Pull request by liminzhu**
Add myget pre-built binaries info on README.

### CODE REVIEW

**Comment by liminzhu** [comment]
@dilijev

**Comment by dilijev** [comment]
Looks fine to me, though I wonder if we can/should add more info about how to get the binaries from those packages. I assume most people don't know they are .zip files with a different extension.

@tcare How do we deal with NuGet packages as a means to distribute pre-built binaries, when they don't follow the NuGet package protocol.

Refer to #85 for work items to improve this.

File: README.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ You can build ChakraCore on Windows 7 SP1 or above, and Windows Server 2008 R2 o
 
 More details in [Building ChakraCore](https://github.com/Microsoft/ChakraCore/wiki/Building-ChakraCore).
 
+Alternatively, you can get pre-built ChakraCore binaries on [myget](https://www.myget.org/feed/chakracore-preview/package/nuget/Microsoft.ChakraCore).
```
Comment (dilijev) [inline README.md]: Change to `pre-built ChakraCore NuGet packages with our latest binaries from [MyGet]`?

**Comment by tcare** [comment]
Agree with Doug on basic instructions to get out the binaries.

As for Doug's concern, I would put a note that it's not in the standard layout and we'll fix that soon.

**Comment by dilijev** [comment]
Maybe we could add a Wiki page about NuGet packages and link to it from the README. Instead of specifically mentioning the MyGet host from the README, we can just say NuGet packages are available and put the links in the Wiki page

This way, we can reduce the amount of content and churn in the README for this purpose. In the short-term, we could explain the structure and how to extract the binaries from the NuGet packages hosted on MyGet. Then, when we fix up the directory structure of the packages we can provide more info on how to integrate those dependencies and develop against them. Later, when we have an official NuGet feed, we can link to that as well.

I'll draft a PR to the Wiki about this and then we can update this PR to point to the right page.

**Comment by liminzhu** [comment]
SGTM :D.

File: README.md (hunk 1 above)
Comment (dilijev) [inline README.md]: In light of https://github.com/Microsoft/ChakraCore-wiki/pull/3 recommend changing this to:

`Alternatively, you can get pre-built ChakraCore binaries from our [NuGet Packages](https://github.com/Microsoft/ChakraCore/wiki/NuGet-Packages).`

**Comment by dilijev** [comment]
@liminzhu FYI I merged in https://github.com/Microsoft/ChakraCore-wiki/pull/3 so you can now link to   https://github.com/Microsoft/ChakraCore/wiki/NuGet-Packages

**Pull request merged**